fala/templates/adviser/results.html

```diff
@@ -11,14 +11,26 @@
   {% if FEATURE_FLAG_SURVEY_MONKEY %}
     {% include 'adviser/_research_banner.html' %}
   {% endif %}
-  <div class="govuk-grid-column-full">
     {% if data.count < 1 %}
+    <div class="govuk-grid-column-full">
         <h1 class="govuk-heading-xl govuk-!-margin-bottom-5">No search results</h1>
         <p class="govuk-body-m" id="no-results-info">There are no results for your criteria.</p>
+    </div>
     {% else %}
-        <h1 class="govuk-heading-xl">Search results</h1>
+    <div class="govuk-grid-column-two-thirds">
+        {% if tailored_results %}
```

Comment:
My thinking that this could grow in size and then we have this in a template that just get bigger and bigger.

What are you're thoughts in putting this into another template and importing it into the results.html. There are example of this already in this template.

Reply:
yeah you make a good point! Happy to pop this code into a template!